tools/clang/lib/SPIRV/CapabilityVisitor.cpp

```diff
@@ -736,27 +736,6 @@ bool CapabilityVisitor::visit(SpirvExtInstImport *instr) {
   return true;
 }
 
-bool CapabilityVisitor::visit(SpirvExtInst *instr) {
-  // OpExtInst using the GLSL extended instruction allows only 32-bit types by
-  // default for interpolation instructions. The AMD_gpu_shader_half_float
-  // extension adds support for 16-bit floating-point component types for these
-  // instructions:
-  // InterpolateAtCentroid, InterpolateAtSample, InterpolateAtOffset
-  if (SpirvType::isOrContainsType<FloatType, 16>(instr->getResultType()))
-    switch (instr->getInstruction()) {
-    case GLSLstd450::GLSLstd450InterpolateAtCentroid:
-    case GLSLstd450::GLSLstd450InterpolateAtSample:
-    case GLSLstd450::GLSLstd450InterpolateAtOffset:
```

Comment:
Do we not generate these instructions anymore? Why can we get rid of this code?

Reply:
The PR description includes the justification, let me know if that seems sufficient

Reply:
Right, sorry I did not read the description closely.